### PR TITLE
Rename Apple TV demo

### PR DESCRIPTION
**Pull request by amlcurran**
## This PR changes

The name of a demo to not include a client's name.

## Considerations and implementation

The demo doesn't actually work. Had to update to the latest Swift 3.0. 

### Test(s) added 

n/a

### Screenshots

n/a

### Paired with 

@ no-one

### CODE REVIEW

File: TVDemo/TVDemo/TVDemo/AppDelegate.swift [hunk 1]
```diff
@@ -12,10 +12,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     static let TVBaseURL = "http://localhost:9001/"
     static let TVBootURL = "\(AppDelegate.TVBaseURL)js/application.js"
 
-    func application(_ application: UIApplication,
-                     didFinishLaunchingWithOptions launchOptions: [NSObject : AnyObject]? = [:]) -> Bool {
-
-        window = UIWindow(frame: UIScreen.main().bounds)
+	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil) -> Bool {
```
Comment (eduardourso) [inline TVDemo/TVDemo/TVDemo/AppDelegate.swift]: is it correctly indented? if you look to the `static lets` and the `funcs` they are not aligned.

File: TVDemo/TVDemo/TV Demo-Bridging-Header.h [hunk 2]
```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
```
Comment (eduardourso) [inline TVDemo/TVDemo/TV Demo-Bridging-Header.h]: do we need this file?

Comment (amlcurran) [inline TVDemo/TVDemo/TV Demo-Bridging-Header.h]: I think so, it told me to make it :/

File: TVDemo/TVDemo/TVDemo.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -65,27 +66,29 @@
...
-		65AE407B1B8C3649006FB795 /* Ch4Demo */ = {
+		65AE407B1B8C3649006FB795 /* Apple TV Demo */ = {
 			isa = PBXGroup;
 			children = (
-				65AE407C1B8C3649006FB795 /* AppDelegate.swift */,
 				65AE40831B8C3649006FB795 /* Assets.xcassets */,
 				65AE40851B8C3649006FB795 /* Info.plist */,
 			);
+			name = "Apple TV Demo";
 			path = Ch4Demo;
```
Comment (FabiaS) [inline TVDemo/TVDemo/TVDemo.xcodeproj/project.pbxproj]: path still says Ch4Demo

File: TVDemo/TVDemo/TVDemo.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -133,9 +136,9 @@
 			);
 			dependencies = (
 			);
-			name = Ch4Demo;
+			name = "TV Demo";
 			productName = Ch4Demo;
```
Comment (FabiaS) [inline TVDemo/TVDemo/TVDemo.xcodeproj/project.pbxproj]: productName still says Ch4Demo

Comment (amlcurran) [inline TVDemo/TVDemo/TVDemo.xcodeproj/project.pbxproj]: I can't find where is this is declared _anywhere_

**Pull request merged**